cmd/app/main.go

```
@@ -1,32 +1,21 @@
-	}
-
-	if err := echo.New(echo.Config{
-		Port:     cfg.Port,
-		Usecases: usecases,
-	}).Start(); err != nil {
+	server, err := app.InitializeEcho(context.Background(), true)
+	if err != nil {
 		panic(err)
 	}
+
+	server.Start()
```

golangci-lint (1.62.2)

Error return value of `server.Start` is not checked

(errcheck)